test6.go

```diff
@@ -1,61 +1,56 @@
 package main
 
 import (
-    "fmt"
-    "io/ioutil"
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
 	"net/http"
 	"os"
-	"log"
-	"io"
-	"encoding/json"
-	"encoding/csv"
 )
 
-type Sheet struct{
-	Sitename string `json:"sitename"`
-	Location string `json:"location"`
-	Address *Address `json:address`
-
+type Sheet struct {
+	Sitename string   `json:"sitename"`
+	Location string   `json:"location"`
+	Address  *Address `json:address`
 }
 
 type Address struct {
 	Number string `json:"number"`
 }
 
-
-
 func main() {
 
 	csvFile, _ := os.Open("test.csv")
-	reader:= csv.NewReader(csvFile)
-	var sheet [] Sheet
+	reader := csv.NewReader(csvFile)
+	var sheet []Sheet
 	for {
-	 line, error:= reader.Read()
-		if error==io.EOF {
+		line, error := reader.Read()
```

Comment:
Error variables are usually named `err`. Also, naming a variable `error` would shadow the `error` interface, which can cause some unexpected bugs on bigger codebases.

Reply:
Ok noted I'll make sure to stick with err